.travis.yml

```diff
@@ -35,10 +36,9 @@ jobs:
         cp -r documentation/docs documentation/docs-tmp
         documentation/bin/createDocu.sh vars documentation/docs-tmp/steps
         docker run --rm -it -v ${TRAVIS_BUILD_DIR}:/docs -w /docs/documentation squidfunk/mkdocs-material:3.0.4 build --clean --verbose --strict
-    - name: Consumer Tests for s4sdk pipeline (CloudFoundry)
-      script: cd consumer-test/s4sdk/CloudFoundry && chmod +x runTests.sh && ./runTests.sh
-    - name: Consumer Tests for s4sdk pipeline (Neo Environment)
-      script: cd consumer-test/s4sdk/NeoEnvironment && chmod +x runTests.sh && ./runTests.sh
+    - name: Consumer Tests
+      if: repo = "SAP/jenkins-library" && ( (type != pull_request && branch =~ /^master$|^it\/.*$/) || (type == pull_request && head_repo = "SAP/jenkins-library" && head_branch =~ /^it\/.*$/) )
```

Comment:
Integration tests are only triggered if no fork is involved. Integration tests are triggered in case the HEAD of the  master branch moves. Integration tests are also triggered if any branch HEAD for branch pattern`it`*` moves.

It is possible to create pull request directly in `it/*`. In this case each update to that pull request results in a integration test build.